src/components/Card/Card.scss

```diff
@@ -20,7 +20,7 @@
   padding: spacing() spacing() 0;
 
   @include page-content-when-not-fully-condensed {
-    padding: spacing(loose) spacing(loose) 0;
+    padding: spacing(loose) spacing(loose) spacing();
```

Comment:
Don’t forget that this change to bottom padding needs to be applied on line 20, for small screens, as well.

In fact, it might be good to write it like this:

```scss
.Header {
  // Shorthand is good for the default breakpoint
  padding: spacing();

  // At other breakpoints, overriding only the specific rules that change makes it easier 
  // to understand intent and can avoid things becoming out of sync across breakpoints
  @include page-content-when-not-fully-condensed {
    padding-top: spacing(loose);
    padding-right: spacing(loose);
    padding-left: spacing(loose);
```